smbios.adoc

```diff
@@ -46,50 +46,49 @@ The current version of the structure is 000Ah (v0.10).
 Bit 7:0 Minor revision +
 The newer revision of RISC-V Processor-specific Block
 Structure is backward compatible with older version of this structure.
-|02h|000Ah|Structure Length|Byte|110|Length of Processor-specific Data
-| 03h| 000Ah| Hart ID| DQWORD| Varies| The ID of this RISC-V Hart
-| 13h| 000Ah| Boot Hart| BYTE| Boolean| 1: This is boot hart to boot system +
+| 02h| 000Ah| Hart ID| DQWORD| Varies| The ID of this RISC-V Hart
```

Comment:
This comment is more about the version. Should we bump the version since we're changing fields?